lambda/translprim.ml

```diff
@@ -127,8 +127,8 @@ let primitives_table =
     "%loc_POS", Loc Loc_POS;
     "%loc_MODULE", Loc Loc_MODULE;
     "%loc_FUNCTION", Loc Loc_FUNCTION;
-    "%field0", Primitive ((Pfield 0), 1);
-    "%field1", Primitive ((Pfield 1), 1);
+    "%field0", Primitive (Pfield(0, Pointer, Mutable), 1);
+    "%field1", Primitive (Pfield(1, Pointer, Mutable), 1);
```

Comment:
Shouldn't another primitive `%field0_imm` (and `%field1_imm`) be added for the definition of `fst` and `snd` which are guaranteed to be immutable ?

Reply:
Please choose the name carefully, at first I read `imm` as meaning "immediate"...

Reply:
Good point. I've had the same confusion before. We use `_imm` in several places in the multicore PR to mean "immediate". One possibility is to use `_immu` for immutable and `_imme` for immediate.